project/api/models/bill.py

```diff
@@ -1,16 +1,24 @@
 from django.db import models
+from django.contrib.postgres.fields import JSONField
 
 
+class BillSources:
+    SUNLIGHT = "sunlight"
+    PROPUBLICA = "propublica"
+    SOURCES = [(SUNLIGHT, SUNLIGHT), (PROPUBLICA, PROPUBLICA)]
+
 class Bill(models.Model):
-    id = models.CharField(max_length=255, primary_key=True) # from sunlight
+    id = models.CharField(max_length=255, primary_key=True)  # from sunlight
 
     title = models.CharField(max_length=1023)
     short_title = models.CharField(max_length=1023)
     short_summary = models.CharField(max_length=1023)
     number = models.IntegerField(default=0)
     b_type = models.CharField(max_length=63)
+    source = models.CharField(max_length=50, choices=BillSources.SOURCES, default=BillSources.SUNLIGHT)
 
-    # status
+    meta = JSONField(default=dict())
```

Comment:
What will be stored in the Meta object? If possible, it would be nice to have explicit fields, to avoid logic to pick apart this object and check for attributes. Also, the structure of the object might change as the third party API changes. So, a generic field like `source_id` might be suffucient here.

Reply:
Oh, thats good point that that thing might change. So let's move the logic to the method and store just urls and id that prolly won't change